input/cql/KidneyHealthEvaluationFHIR.cql

```diff
@@ -79,12 +86,18 @@ define "Has Outpatient Visit During Measurement Period":
         and ValidEncounter.status = 'finished'
   )
 
+/*
+@QICore6Uplift: Update Condition to ConditionProblemsHealthConcerns
+*/ 
```

Comment:
As written this should also be a union of ConditionProblemsHealthConcerns and ConditionEncounterDiagnosis

Reply:
Updated comment to be a union of ConditionProblemsHealthConcerns and ConditionEncounterDiagnosis.